### PR TITLE
delete button for templates

### PR DESCRIPTION
**Pull request by rosasck**
Deleting a response from the table is complete and tested

### CODE REVIEW

File: simcon_project/users/views/researcher_home.py [hunk 1]
```diff
@@ -1,10 +1,16 @@
 from django.shortcuts import render
 from conversation_templates.models import TemplateResponse
 from django.contrib.auth.decorators import login_required
+from django.core.paginator import Paginator
 
 
 @login_required(login_url="/accounts/login/")
 def ResearcherView(request):
     responseTable = TemplateResponse.objects.all()
```
Comment (devin-w-hill) [inline simcon_project/users/views/researcher_home.py:9]: Do we want every researcher to be able to see and delete all responses? Seems like we should filter based on whatever they have assigned. Unrelated to the ticket though

**Review by devin-w-hill — APPROVED** [review]
Works great! It deleted the submission from the database. I pointed out a side-issue, maybe to fix in another ticket.

File: simcon_project/users/views/researcher_home.py (hunk 1 above)
Comment (rosasck) [reply simcon_project/users/views/researcher_home.py:9]: we don't have an actual field that relates them back.... i just realized my assumption was wrong about this table only holds there responses

**Review by Likhovodov — APPROVED** [review]
Nice

**Pull request merged**